src/app/components/LiveHeaderMedia/index.styles.tsx

```diff
@@ -131,9 +134,6 @@ export default {
       display: 'block',
       width: '100%',
       span: { margin: 0 },
-      'button:hover & span, button:focus & span': {
```

Comment:
We need this in the close state e.g. text underline on the media title and brand